Source/DafnyLanguageServer/Workspace/TextDocumentLoader.cs

```diff
@@ -78,18 +81,17 @@ ILoggerFactory loggerFactory
       );
     }
 
-    public Task<DafnyDocument> LoadAsync(TextDocumentItem textDocument, CancellationToken cancellationToken) {
+    public async Task<DafnyDocument> LoadAsync(TextDocumentItem textDocument, CancellationToken cancellationToken) {
 #pragma warning disable CS1998
-      // By using `async`, any OperationCancelledExceptions are converted to a cancelled Task.
-      return Task.Factory.StartNew(async () => LoadInternal(textDocument, cancellationToken), cancellationToken,
-        TaskCreationOptions.None, LargeStackScheduler).Unwrap();
+      return await await Task.Factory.StartNew(async () => LoadInternal(textDocument, cancellationToken), cancellationToken,
 #pragma warning restore CS1998
+        TaskCreationOptions.None, LargeStackScheduler);
     }
 
     private DafnyDocument LoadInternal(TextDocumentItem textDocument, CancellationToken cancellationToken) {
       var errorReporter = new DiagnosticErrorReporter(textDocument.Uri);
       var program = parser.Parse(textDocument, errorReporter, cancellationToken);
-      PublishDafnyLanguageServerLoadErrors(errorReporter, program);
+      IncludePluginLoadErrors(errorReporter, program);
```

Comment:
Why "include" instead of "publish"?

Reply:
Publish is terminology used for sending diagnostics to the LSP clients (https://microsoft.github.io/language-server-protocol/specification#textDocument_publishDiagnostics), which is not yet being done here.

Reply:
Good point, then maybe "collect"?  But this is minor.